sources/packages/api/src/route-controllers/education-program-offering/models/education-program-offering.dto.ts

```diff
@@ -0,0 +1,61 @@
+export interface CreateEducationProgramOfferingDto {
+  name: string;
+  studyStartDate: Date;
+  studyEndDate: Date;
+  breakStartDate: Date;
+  breakEndDate: Date;
+  actualTuitionCosts: number;
+  programRelatedCosts: number;
+  mandatoryFees: number;
+  exceptionalExpenses: number;
+  tuitionRemittanceRequestedAmount: number;
+  offeringDelivered: string;
+  lacksStudyDates: boolean;
+  lacksStudyBreaks: boolean;
+  lacksFixedCosts: boolean;
+  tuitionRemittanceRequested: string;
+}
+
+export interface UpdateEducationProgramOfferingDto {
```

Comment:
Do we need 2 DTOs with exactly the same fields? I believe that they will evolve together and we can keep only one.

Reply:
done